src/test/kotlin/xyz/block/dap/examples/DapResolveMoneyAddressesExample.kt

```diff
@@ -0,0 +1,25 @@
+package xyz.block.dap.examples
```

Comment:
I wonder if this should be in the main source, instead of the test?

Reply:
We could also create a project module and keep it separate

Reply:
I'll commit this as is for now, but look into that.